sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/SampleRateTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Azure.Monitor.OpenTelemetry.Exporter.Integration.Tests.TestFramework;
+using Azure.Monitor.OpenTelemetry.Exporter.Internals;
+using Azure.Monitor.OpenTelemetry.Exporter.Models;
+using OpenTelemetry;
+using OpenTelemetry.Extensions.AzureMonitor;
+using OpenTelemetry.Trace;
+using Xunit;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Tests
+{
+    public class SampleRateTests
+    {
+        private const string ActivitySourceName = "SampleRateTests";
+        private const string ActivityName = "TestActivity";
+
+        static SampleRateTests()
+        {
+            Activity.DefaultIdFormat = ActivityIdFormat.W3C;
+            Activity.ForceDefaultIdFormat = true;
+
+            var listener = new ActivityListener
```

Comment:
its a bit odd that we have to setup own listeners? Can we use OTel SDK TracerProvider instead with AlwaysOnSampler?

Reply:
All our tests follow this model, we might need to refactor all of the tests in a different task.

Reply:
These tests were based on old OTel approach and needs an upgrade.